### DISCUSSION TITLE
Date/Time Formats

### DISCUSSION THREAD

**Discussion by dstark** [Extended-Thunder/send-later#528] [General]
Would you be able to point me to any documentation about how to use the custom date/time formats in the preferences section? I've tried multiple different formats (e.g., "d Mmm yy") but only then get the letters—not the date and time information they should represent—back out in the Send Later column of the drafts folder.

![image](https://user-images.githubusercontent.com/205528/224712953-a1d048ea-53c4-48ce-b01a-1fb063d965a8.png)

Thank you so much!

**Comment by jikamens** [comment]
Yeah, we need to document this. Tracking in #566.

**Comment by jikamens** [comment]
This is done. See https://extended-thunder.github.io/send-later/#date-format-problems which explains why you might want to customize the date formats if Send Later isn't working properly, and https://extended-thunder.github.io/send-later/#custom-dates which explains what the custom date format strings look like (well, actually, to be honest, it just refers you to https://sugarjs.com/dates/#/Formatting, but you should still read the section in the user guide because there's an important caveat there you should be aware of).

**Reply by dstark** [reply]
Noted. Thanks so much!
